mpf/devices/multiball.py

```diff
@@ -291,6 +291,54 @@ def add_a_ball(self):
             self.balls_added_live += 1
             self.machine.game.balls_in_play += 1
             self.source_playfield.add_ball(balls=1)
+            self.add_a_ball_timer_start()
+
+    def add_a_ball_timer_start(self) -> None:
```

Comment:
guess this should be `_add_a_ball_timer_start` as it should be private and not called from outside this class.

Reply:
Agreed, and fixed in my next push.

Reply:
While I am in this file.  The timer_start method for the regular ball save is not private.  Do we want it to be?  If so, I'll make that change as well in my push.

Reply:
Good find. I would make that private as well.